modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
+.Prerequisites
+
+* You must have administrator privileges.
```

Comment:
I also need access to whatever platform I can perform these actions on, correct?

Reply:
Same as above.

Reply:
Need more context here @deerskindoll.